README.md

```diff
@@ -32,16 +32,17 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? Semantic HTML would be using semantic tags that let the developer or the browser know what its designating a certain styling. 
 
```

Comment:
Very true, semantic code also helps with SEO and accessibility. ->https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML